### PR TITLE
Project Building Blocks contribution

### PR DESCRIPTION
**Pull request by Servante**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

**Comment by Javier-Machin** [comment]
HI @Servante 

Please remove all the lines generated by the conflict.

Thanks

File: ruby_programming/basic_ruby/project_building_blocks.md [hunk 1]
```diff
@@ -35,9 +35,13 @@ Submit a link below to this [file](https://github.com/TheOdinProject/curriculum/
 
 
 * Add your solution below this line!
+<<<<<<< HEAD
```
Comment (Javier-Machin) [inline ruby_programming/basic_ruby/project_building_blocks.md]: Remove this

**Comment by Servante** [comment]
Thanks Javier, I think I got everything fixed. Sorry, new user here.

**Comment by Javier-Machin** [comment]
Don't worry @Servante , good job and thanks for fixing it 👍.

**Pull request merged**